model/src/w3iogrmd.F90

```diff
@@ -545,15 +546,15 @@ SUBROUTINE W3IOGR ( INXOUT, NDSM, IMOD, FEXT )
 !
 ! open file ---------------------------------------------------------- *
 !
-      I      = LEN_TRIM(FILEXT)
-      J      = LEN_TRIM(FNMPRE)
+      IEXT   = LEN_TRIM(FILEXT)
+      IPRE   = LEN_TRIM(FNMPRE)
 !
-!AR: ADD DEBUGFLAG      WRITE(*,*) 'FILE=', FNMPRE(:J)//'mod_def.'//FILEXT(:I)
+!AR: ADD DEBUGFLAG      WRITE(*,*) 'FILE=', FNMPRE(:IPRE)//'mod_def.'//FILEXT(:IEXT)
       IF ( WRITE ) THEN
-          OPEN (NDSM,FILE=FNMPRE(:J)//'mod_def.'//FILEXT(:I),         &
+          OPEN (NDSM,FILE=FNMPRE(:IPRE)//'mod_def.'//FILEXT(:IEXT),   &
```

Comment:
I don't understand why the much simpler form of 
```
OPEN (NDSM,FILE=trim(FNMPRE)//'mod_def.'//trim(FILEXT)
```
isn't utilized. Why bother declaring two local variables to record the string length when the intrinsic ``trim`` can do what you need?

Reply:
@DeniseWorthen According to the commit message:
w3iogrmd.F90: add variables to avoid redefinition of I later on

Reply:
Yes, I understand that. My point none of the definitions for a local string length (either the original or the changed) are necessary if you simply use the intrinsic trim function on the character string.

Reply:
The point was to minimize changes in case there was some side effect I did not know about. I agree TRIM is probably the cleaner way to do it.